src/App.tsx

```diff
@@ -1,61 +1,137 @@
+/* eslint-disable no-console */
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('0');
+  const [visibleTodos, setVisibleTodos] = useState(todosFromServer);
+  const [errorTitle, setErrorTitle] = useState(false);
```

Comment:
Let's name it titleError. As errorTitle sounds like it should be a text not a boolean.